examples/example-ui-react/src/MySvelteComponent/react.ts

```diff
@@ -1,13 +1,14 @@
-import SvelteToReact from "../../../../web-components/svelte-react";
+import SvelteToReact from '../../../../web-components/svelte-react'
 import Button from './button.svelte'
 
 // Prop types for a svelte component are not able to be inferred at compile-time,
 // only at dev time in typescript plugins, so we must duplicate it here, manually.
 // TODO: have this type definition be auto-generated in the bundler.
 
-export interface HelloEvent extends CustomEvent<{
-  text: string
-}> {}
+export interface HelloEvent
```

Comment:
IMO this example is less readable, but that's minor!

Reply:
Yup, I tend to agree.